packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasEstimates.ts

```diff
@@ -1,29 +1,27 @@
 import { BigNumber, Signer } from 'ethers'
-import { useMemo } from 'react'
 import useSWR from 'swr'
 import { useSigner } from 'wagmi'
-import { Provider } from '@ethersproject/providers'
 
 import { DepositGasEstimates, GasEstimates } from '../arbTokenBridge.types'
 import { BridgeTransferStarterFactory } from '@/token-bridge-sdk/BridgeTransferStarterFactory'
-import { getProviderForChainId } from '../useNetworks'
 
 async function fetcher([
   signer,
-  sourceChainProvider,
-  destinationChainProvider,
+  sourceChainId,
```

Comment:
Gas estimation code runs quite frequently and keeps on re-initializing the bridge-transfer-starter. Hence, we should optimize the RPC calls here. Having chain-ids instead of providers to initialize the bridge saves calls.
We need chain-ids in factory to figure out what kind of transfer it is, but if we pass chain-ids as props, we avoid figuring those out from providers.
Figuring out providers from chain-ids (Reverse) doesn't incur any RPC calls.